_includes/articles/cardcoins-rbf-batching.md

```diff
@@ -0,0 +1,48 @@
+{:.post-meta}
+*by [CardCoins][]*
+
+[Replace By Fee][topic rbf] (RBF, BIP-125) and [batching][payment batching] are two important tools for any enterprises
```

Comment:
Nit: use our normal BIP styling; the consistency is especially helpful to me when I'm `git grep`ing for things we previously wrote.

```suggestion
[Replace By Fee][topic rbf] (RBF, BIP125) and [batching][payment batching] are two important tools for any enterprises
```

Reply:
Also needs edit later in the article (not important for git grep, but important for intra-article consistency).